src/ppx/grfx/vk/vk_pipeline.cpp

```diff
@@ -172,22 +173,22 @@ Result GraphicsPipeline::InitializeVertexInput(
     }
 
     //// Fill out Vulkan attributes and bindings
-    //for (auto& binding : mInputBindings) {
-    //    for (auto& attribute : binding.attributes) {
-    //        VkVertexInputAttributeDescription vkAttribute = {};
-    //        vkAttribute.location                          = attribute.location;
-    //        vkAttribute.binding                           = attribute.binding;
-    //        vkAttribute.format                            = ToVkFormat(attribute.format);
-    //        vkAttribute.offset                            = attribute.offset;
-    //        vkAttributes.push_back(vkAttribute);
-    //    }
+    // for (auto& binding : mInputBindings) {
```

Comment:
maybe remove this dead code? (since it's getting formatted)

Reply:
Done.